proto/frequenz/api/common/v1/microgrid/components/components.proto

```diff
@@ -141,6 +142,21 @@ message Component {
 
   // The operational lifetime of the component.
   frequenz.api.common.v1.microgrid.Lifetime operational_lifetime = 9;
+
+  // List of rated bounds present for the component identified by Metric.
+  repeated MetricRatedBounds rated_bounds = 10;
+}
+
+// Defines rated bound for a specific metric.
+message MetricRatedBounds {
+  // Metric type the rated bounds are for
+  frequenz.api.common.v1.metrics.Metric metric = 1;
+
+  // The rated bounds of the specified metric.
+  //
+  // This contains the lower and upper bounds for said metric as specified
+  // by the component manufacturer's ratings.
+  frequenz.api.common.v1.metrics.Bounds rated_bounds = 2;
```

Comment:
After a brief discussion with @thomas-nicolai-frequenz , it is now clearer that this is not just the manufacturer's rated bounds, but a more general pair of static bounds. What this means is that these bounds can be populated by combining multiple sources.

E.g., for min/max AC power for inverters, two inputs would be required:
a. The fuse limits of the inverter
b. The manufacturer limits of the inverter
The final static bounds would then be the intersection of these two inputs.

So, `rated_bounds` is not an accurate name for this. Perhaps `config_bounds`/`MetricConfigBounds`?
Any other alternatives?

Reply:
I like `config_bounds` & `MetricConfigBounds`, maybe `static_bounds` & `MetricStaticBounds` might be appropriate too so long as we can count on not these not changing without some event that we would restart the core apps for.  `config_bounds` works for me though.

Reply:
I still feel we need to properly explain/document what a "MetricBound" is. Its more that these bounds limit something like power output and it happens that power is expressed in a certain metric. Metrics itself don't have bounds. You know what I mean?

Reply:
>I still feel we need to properly explain/document what a "MetricBound" is. Its more that these bounds limit something like power output and it happens that power is expressed in a certain metric. Metrics itself don't have bounds. You know what I mean?

We can extend the documentation. Would `ComponentConfigBounds` be more suitable name?

Reply:
> Would ComponentConfigBounds be more suitable name?

I think `MetricConfigBounds` might make a bit more sense, since these are only being used in `Component`.

Having component bounds within the component wouldn't describe what the bounds are for in relation to the component; whereas `MetricConfigBounds` would be a bit more descriptive in that regard.
If we extend the usage beyond just `Component` the name would no longer be descriptive.

That being said I'm Ok with either, I think they both would work in this context.

Reply:
Then `MetricConfigBounds` it is :)